### ISSUE TITLE
Website

### ISSUE THREAD

**Issue by CanadianKunukJosh** [LeeTwentyThree/SubnauticaModManager#24]
Is the website down because i can't access any of the mods that I use to have last month

**Comment by LeeTwentyThree** [comment]
Yes, it's down. We're looking into fixing the problem but it won't be easy. For now we can help you find mods in the Subnautica modding server.

**Comment by CanadianKunukJosh** [comment]
Oh ok that's quite helpful, thank you. Ill try and get into the modding server here now

**Comment by ModderMRT** [comment]
is the submodica website down also all the links attached to the submodica web page from discord are also not working

**Comment by LeeTwentyThree** [comment]
Yes, Submodica was the source of the mod manager’s mods.
